### ISSUE TITLE
Missing mouse pup calls, any way to enhance detection?

### ISSUE THREAD

**Issue by t-chari** [DrCoffey/DeepSqueak#59]
Hi,

I've been trying to retrain the existing mouse call network to detect pup calls since the original network is missing more than half of the calls in my files compared to manual counting. I tried to retrain the network with around 20 3-minute audio files, but it it seems like it's still missing a large chunk of the calls. Is there any way to modify the sensitivity to allow the network to recognize the calls that are missing? Or do I need to feed more files into the network in order for it to recognize these calls that it's missing?

Thanks and have a good weekend!
Trish

**Comment by DrCoffey** [comment]
Hey Trish,

I'm not too familiar with pup calls, would you mind sharing an audio file with some calls so I can better help you out?

-Kevin

**Comment by t-chari** [comment]
Hi Kevin,

Sure, how should I go about sending you a .wav file? I actually tried retraining the network using high recall and now it is able to detect more calls though there are some files where some calls are still being missed.

Thanks!
Trish

**Comment by DrCoffey** [comment]
A google drive or dropbox link would work

**Comment by t-chari** [comment]
Great! Here is a google drive link: https://drive.google.com/open?id=1XnYmGoyzMXyhWSIW74_qVrJ9JERdrAEq

There will be some noticeable noise around 40 kHz (this was due to noise from a light motion sensor that we've now fixed), but the network was pretty good at not recognizing it as a call!

Thanks,
Trish

**Comment by DrCoffey** [comment]
Hey,

I took a look at that file and I think it is just very quiet. You have a lot of range left to play with, so I would turn up the gain on future recordings.

I was able to get pretty good detection using the short rat call network with the middle sensitivity.

-Kevin

**Comment by t-chari** [comment]
Hi Kevin, 

Thanks for the reply! Since we last spoke, I actually was able to enhance the call detection by using one of my trained networks with high recall and retraining it once I eliminated all the noisy calls. 

I've been using Avisoft CM16/CMPA microphone. The company recommended me an optimal range for the gain and I'm currently at the max of that range. I could try testing even higher gain on future files!

I've also been trying to classify the calls using unsupervised classification and then supervised clustering after rejecting incorrectly classified calls. I've been trying to classify them based on 10 call types traditionally used for pup USVs (ex. harmonic, chevron, downward/upward). I was wondering what the best parameters would be for the unsupervised classification? I increased the weight for the shape and frequency since that's how these pup USVs are usually classified, but I wanted your input on how to best modify the weights and I'm unsure of the optimal settings for max clusters and repetitions. 

Thanks for your help! DeepSqueak has been immensely useful!

Trish
